packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -37,10 +37,47 @@ const propTypes = {
    * Index within the Menu Stack.
    */
   index: PropTypes.number.isRequired,
+  /**
+   * Bounding container for the menu, will use window if no value provided.
+   */
+  boundingRef: PropTypes.func,
+  /**
+   * Indicates if the menu content should set default focus on itself.
+   */
+  isFocused: PropTypes.bool,
+  /**
+   * Indicates if menu's height has been constrained by bounding container.
+   */
+  isHeightBounded: PropTypes.bool,
+  /**
+   * Indicates if menu's width has been constrained by bounding container.
+   */
+  isWidthBounded: PropTypes.bool,
+  /**
+   * Fixed height for content.
+   */
+  fixedHeight: PropTypes.number,
+  /**
+   * Fixed width for content.
+   */
+  fixedWidth: PropTypes.number,
+  /**
+   * Width for content.
+   */
+  contentWidth: PropTypes.number,
+  /**
+   * Indicates if the content should be hidden.
+   */
+  isHidden: PropTypes.bool,
+  /**
+   * Ref callback function to be applied to content container.
+   */
+  refCallback: PropTypes.func,
 };
 
 const defaultProps = {
```

Comment:
Do we need to add default values of false for isHidden, isHeightBounded, and isWidthBounded?

Reply:
Added here: https://github.com/cerner/terra-core/pull/942/commits/1c7fac345b5bd12453513ffe3a54b4fd568dc5f5